docs/admin/pool_operations.md

```diff
@@ -1,38 +1,124 @@
+  This includes draining, excluding and re-integrating targets,
+  and evicting client connections to a pool.
+
+* An upgrade command to upgrade a pool's format version
+  after a DAOS software upgrade.
+
 ### Creating a Pool
 
-A DAOS pool can be created and destroyed through the `dmg` utility.
+A DAOS pool can be created through the `dmg pool create` command.
+The mandatory parameters that are needed for the creation of a pool
+are the pool label, and a specification of the size of the storage
+allocation.
+
+The pool label must consist of alphanumeric characters, colon (`:`),
+period (`.`), hyphen (`-`) or underscore (`_`).
+The maximum length of a pool label is 127 characters.
+Labels that can be parsed as a UUID (e.g. 123e4567-e89b-12d3-a456-426614174000)
+are forbidden. Pool labels must be unique across the DAOS system.
+
+A pool's size is determined by two factors: How many storage engines are
+participating in the storage allocation, and how much capacity in each
+storage tier is allocated (the latter can be specified either on a per-engine
+basis, or as the total for the pool across all participating engines).
+The same amount of storage will be allocated on each of the participating
+storage engines. If one or more of those engines do not have sufficient free
+space for the requested capacity, the pool creation will fail.
+
+If neither the `--nranks` nor the `--ranks` option is used,
+then the pool will span all storage engines of the DAOS system.
+To limit the pool to only a subset of the engines, those two options
+can be used to specify either the desired number of engines,
+or an explicit list of engine ranks to be used for this pool.
+
+The capacity of the pool can be specified in three different ways:
+
+1. The `--scm-size` parameter (and optionally `--nvme-size`) can
+   be used to specify the SCM capacity (and optionally the NVMe
+   capacity) _per storage engine_ in **Bytes**.
+   The minimum SCM size is 16 MiB per **target**, so for a storage
+   engine with 16 targets the minimum is `--scm-size=256MiB`.
+   The NVMe size can be zero. If it is non-zero then the minimum
+   NVMe size is 1 GiB per **target**, so for a storage engine
+   with 16 targets the minimum non-zero NVMe size is
+   `--nvme-size=16GiB`.
+   To derive the total pool capacity, these per-engine capacities
+   have to be multiplied by the number of participating engines.
+
+2. The `--size` option can be used to specify the _total_ pool
+   capacity in **Bytes**. This value denotes the sum of the SCM
+   and NVMe capacities. The relative contributions of the SCM and
+   NVMe storage tiers to the total pool size are determined by the
+   `--tier-ratio` parameter.
+   By default this ratio is `6,94`, so for a pool of size 100TB
+   there will be 6TB of SCM and 94 TB of NVMe storage.
+   An SCM-only pool can be created by using `--tier-ratio 100,0`.
+
+3. The `--size` option can be used to specify the _total_ pool
```

Comment:
Maybe we could mention that the current implementation do not support rank filtering.

Reply:
i didn't mention that intentionally, as i'm trying to get that resolved in DAOS-10885 (and it's not the topic of DAOS-10039 that this PR is addressing). there's a clear error message if you try, so i think not documenting this is ok here.

Reply:
There is already a JIRA ticket for this limitation:
https://daosio.atlassian.net/browse/DAOS-9557
From my side, this feature should be easily and quickly added to the current version of the code.
Sadly, I did not  yet have the time to work on it.

Otherwise, I am ok with your arguments for not integrating this limitation in the user guide.